bbb-common-message/src/main/scala/org/bigbluebutton/common2/msgs/VoiceConfMsgs.scala

```diff
@@ -190,6 +190,16 @@ case class MuteUserCmdMsg(
 ) extends StandardMsg
 case class MuteUserCmdMsgBody(userId: String, mutedBy: String, mute: Boolean)
 
+/**
+ * Sent by client to set input id listen only value for user in the voice conference.
+ */
+object SetUserListenOnlyInputCmdMsg { val NAME = "SetUserListenOnlyInputCmdMsg" }
+case class SetUserListenOnlyInputCmdMsg(
+    header: BbbClientMsgHeader,
+    body:   SetUserListenOnlyInputCmdMsgBody
+) extends StandardMsg
+case class SetUserListenOnlyInputCmdMsgBody(userId: String, listenOnlyInputDevice: Boolean)

```

Comment:
I think it's not being used:

```suggestion
case class SetUserListenOnlyInputCmdMsgBody(listenOnlyInputDevice: Boolean)
```